packages/web/graphql/mutations/idxCache.ts

```diff
@@ -1,7 +1,5 @@
 export const InsertCacheInvalidation = /* GraphQL */ `
   mutation InsertCacheInvalidation($playerId: uuid!) {
-    updateIDXProfile(playerId: $playerId) {
-      success
-    }
+    updateIDXProfile(playerId: $playerId)
```

Comment:
I think @alalonde had added the success param here?

Reply:
I switched this method to asynchronous to speed it up. It now returns a UUID that can be used to query for it's final value. That query method is in `schema.graphql`.

I originally had this as:
```graphql
    updateIDXProfile(playerId: $playerId) {
      updateIDXProfile: uuid
    }
```
But I get an error: “`GraphQLDocumentError: Field "updateIDXProfile" must not have a selection since type "uuid!" has no subfields.`” I'm pretty sure what I have is how you designate it returns a single field.

Reply:
Ah i see. Thanks for clarifyng this! :D